bridges/EconomistBridge.php

```diff
@@ -0,0 +1,64 @@
+<?php
+class EconomistBridge extends BridgeAbstract {
+	const NAME = 'The Economist: Latest Updates';
+	const URI = 'https://www.economist.com';
+	const DESCRIPTION = 'Fetches the latest updates from the Economist.';
+	const MAINTAINER = 'thefranke';
+	const CACHE_TIMEOUT = 60 * 60; // 1h
```

Comment:
Since this is a const and the comment already mentions the total duration I think it makes sense to simply put the actual value here.

```suggestion
	const CACHE_TIMEOUT = 3600; // 1h
```